go/parquet/cmd/parquet_reader/Makefile

```diff
@@ -0,0 +1,6 @@
+PROG_NAME := "parquet_reader"
+VERSION = 0.1.$(shell date -u +%Y%m%d.%H%M)
```

Comment:
This version should probably reflect the Arrow/Parquet version rather than being `0.1.<date>`, can we use something like: `TAG=$(git name-rev --tags --name-only $(git rev-parse HEAD)) && TAG=${TAG%^0} && TAG=${TAG#apache-arrow-} && echo $TAG` which will get the current tag and remove the `apache-arrow-` prefix to get the version on it's own? then combine it with the date.